### PR TITLE
Update timeline.go

### PR DESCRIPTION
**Pull request by jundow**
Add initialization of ev.Tags before parsing on doPost function in timeline.go

### CODE REVIEW

**Comment by jundow** [comment]
FYI, the comparison of the result between before and after the change.
Obtained by fmt.Println right after each event was published to the corresponding relay.

```
$ ./algia.exe post "Test Tags: nil"
------
relay: wss://relay.nostr.wirednet.jp
status: success
err: <nil>
ev: {"id":"68f3e2d556e9bd68021cc0df3db5df3477ace51325fde890ffed4e140f5456a4","pubkey":"9fe72c76ced19360f2e62d89b8b54f80fdea877a1f334b58b1e4bdf1e3a5f902","created_at":1684101025,"kind":1,"tags":null,"content":"Test Tags: nil","sig":"a23977ce476f17fe3a23d5638f31d6943d44fe8bbb5d4153ab7806f9d7dfac11fe69f0da43ff473bd9083cea704112aa86ab19989ea10bc7e1e5fcecee6708f1"}
------
relay: wss://nos.lol
status: failed
err: msg: invalid: std::get: wrong index for variant
ev: {"id":"68f3e2d556e9bd68021cc0df3db5df3477ace51325fde890ffed4e140f5456a4","pubkey":"9fe72c76ced19360f2e62d89b8b54f80fdea877a1f334b58b1e4bdf1e3a5f902","created_at":1684101025,"kind":1,"tags":null,"content":"Test Tags: nil","sig":"a23977ce476f17fe3a23d5638f31d6943d44fe8bbb5d4153ab7806f9d7dfac11fe69f0da43ff473bd9083cea704112aa86ab19989ea10bc7e1e5fcecee6708f1"}
------
relay: wss://relay.nostr.band
status: success
err: <nil>
ev: {"id":"68f3e2d556e9bd68021cc0df3db5df3477ace51325fde890ffed4e140f5456a4","pubkey":"9fe72c76ced19360f2e62d89b8b54f80fdea877a1f334b58b1e4bdf1e3a5f902","created_at":1684101025,"kind":1,"tags":null,"content":"Test Tags: nil","sig":"a23977ce476f17fe3a23d5638f31d6943d44fe8bbb5d4153ab7806f9d7dfac11fe69f0da43ff473bd9083cea704112aa86ab19989ea10bc7e1e5fcecee6708f1"}
------
relay: wss://relay-jp.nostr.wirednet.jp
status: sent
err: <nil>
ev: {"id":"68f3e2d556e9bd68021cc0df3db5df3477ace51325fde890ffed4e140f5456a4","pubkey":"9fe72c76ced19360f2e62d89b8b54f80fdea877a1f334b58b1e4bdf1e3a5f902","created_at":1684101025,"kind":1,"tags":null,"content":"Test Tags: nil","sig":"a23977ce476f17fe3a23d5638f31d6943d44fe8bbb5d4153ab7806f9d7dfac11fe69f0da43ff473bd9083cea704112aa86ab19989ea10bc7e1e5fcecee6708f1"}
------
relay: wss://nostr-relay.nokotaro.com
status: sent
err: <nil>
ev: {"id":"68f3e2d556e9bd68021cc0df3db5df3477ace51325fde890ffed4e140f5456a4","pubkey":"9fe72c76ced19360f2e62d89b8b54f80fdea877a1f334b58b1e4bdf1e3a5f902","created_at":1684101025,"kind":1,"tags":null,"content":"Test Tags: nil","sig":"a23977ce476f17fe3a23d5638f31d6943d44fe8bbb5d4153ab7806f9d7dfac11fe69f0da43ff473bd9083cea704112aa86ab19989ea10bc7e1e5fcecee6708f1"}
------
relay: wss://nostr.h3z.jp
status: sent
err: <nil>
ev: {"id":"68f3e2d556e9bd68021cc0df3db5df3477ace51325fde890ffed4e140f5456a4","pubkey":"9fe72c76ced19360f2e62d89b8b54f80fdea877a1f334b58b1e4bdf1e3a5f902","created_at":1684101025,"kind":1,"tags":null,"content":"Test Tags: nil","sig":"a23977ce476f17fe3a23d5638f31d6943d44fe8bbb5d4153ab7806f9d7dfac11fe69f0da43ff473bd9083cea704112aa86ab19989ea10bc7e1e5fcecee6708f1"}



$ ./algia.exe post "Test Tags: []"
------
relay: wss://nostr-relay.nokotaro.com
status: success
err: <nil>
ev: {"id":"25ddc462415027ca70c217f1e5ecbc098de2ce6d4a157c4c05d52eab725f10e6","pubkey":"9fe72c76ced19360f2e62d89b8b54f80fdea877a1f334b58b1e4bdf1e3a5f902","created_at":1684101145,"kind":1,"tags":[],"content":"Test Tags: []","sig":"e0e5eb2a22d2daf1cce779dd6495b108e8b66ff9ca0bb23f1d29940260f82ebeef367a754465768206ae0e47d186ac00c40b99ddaa6c06e4fa24c18010ebdeea"}
------
relay: wss://nostr.h3z.jp
status: success
err: <nil>
ev: {"id":"25ddc462415027ca70c217f1e5ecbc098de2ce6d4a157c4c05d52eab725f10e6","pubkey":"9fe72c76ced19360f2e62d89b8b54f80fdea877a1f334b58b1e4bdf1e3a5f902","created_at":1684101145,"kind":1,"tags":[],"content":"Test Tags: []","sig":"e0e5eb2a22d2daf1cce779dd6495b108e8b66ff9ca0bb23f1d29940260f82ebeef367a754465768206ae0e47d186ac00c40b99ddaa6c06e4fa24c18010ebdeea"}
------
relay: wss://relay.nostr.wirednet.jp
status: success
err: <nil>
ev: {"id":"25ddc462415027ca70c217f1e5ecbc098de2ce6d4a157c4c05d52eab725f10e6","pubkey":"9fe72c76ced19360f2e62d89b8b54f80fdea877a1f334b58b1e4bdf1e3a5f902","created_at":1684101145,"kind":1,"tags":[],"content":"Test Tags: []","sig":"e0e5eb2a22d2daf1cce779dd6495b108e8b66ff9ca0bb23f1d29940260f82ebeef367a754465768206ae0e47d186ac00c40b99ddaa6c06e4fa24c18010ebdeea"}
------
relay: wss://relay-jp.nostr.wirednet.jp
status: success
err: <nil>
ev: {"id":"25ddc462415027ca70c217f1e5ecbc098de2ce6d4a157c4c05d52eab725f10e6","pubkey":"9fe72c76ced19360f2e62d89b8b54f80fdea877a1f334b58b1e4bdf1e3a5f902","created_at":1684101145,"kind":1,"tags":[],"content":"Test Tags: []","sig":"e0e5eb2a22d2daf1cce779dd6495b108e8b66ff9ca0bb23f1d29940260f82ebeef367a754465768206ae0e47d186ac00c40b99ddaa6c06e4fa24c18010ebdeea"}
------
relay: wss://nos.lol
status: success
err: <nil>
ev: {"id":"25ddc462415027ca70c217f1e5ecbc098de2ce6d4a157c4c05d52eab725f10e6","pubkey":"9fe72c76ced19360f2e62d89b8b54f80fdea877a1f334b58b1e4bdf1e3a5f902","created_at":1684101145,"kind":1,"tags":[],"content":"Test Tags: []","sig":"e0e5eb2a22d2daf1cce779dd6495b108e8b66ff9ca0bb23f1d29940260f82ebeef367a754465768206ae0e47d186ac00c40b99ddaa6c06e4fa24c18010ebdeea"}
------
relay: wss://relay.nostr.band
status: success
err: <nil>
ev: {"id":"25ddc462415027ca70c217f1e5ecbc098de2ce6d4a157c4c05d52eab725f10e6","pubkey":"9fe72c76ced19360f2e62d89b8b54f80fdea877a1f334b58b1e4bdf1e3a5f902","created_at":1684101145,"kind":1,"tags":[],"content":"Test Tags: []","sig":"e0e5eb2a22d2daf1cce779dd6495b108e8b66ff9ca0bb23f1d29940260f82ebeef367a754465768206ae0e47d186ac00c40b99ddaa6c06e4fa24c18010ebdeea"}

```

File: timeline.go [hunk 1]
```diff
@@ -257,6 +257,7 @@ func doPost(cCtx *cli.Context) error {
 		return errors.New("content is empty")
 	}
 
+        ev.Tags = nostr.Tags{ }
 	for i, u := range cCtx.StringSlice("u") {
```
Comment (mattn) [inline timeline.go:261]: please do `go fmt`

**Review by mattn — APPROVED** [review]
Thank your contribution

File: timeline.go [hunk 2]
```diff
@@ -257,6 +257,7 @@ func doPost(cCtx *cli.Context) error {
 		return errors.New("content is empty")
 	}
 
+        ev.Tags = nostr.Tags{ }
```
Comment (mattn) [inline timeline.go]:
```suggestion
	ev.Tags = nostr.Tags{ }
```

Comment (jundow) [reply timeline.go]: Please forgive me my silly question, but is there anything I have to next?

I requested a change but I have almost zero knowledge about GitHub's workflow and I do not understand whether if I was expected to do something for the next.

Thanks.

**Comment by mattn** [comment]
Thank you

**Pull request merged**